data/compliance/pages.json

```diff
@@ -0,0 +1,97 @@
+[
+    {
+        "framework": "PCI DSS",
+        "cloud": "AWS",
+        "description": "PCI DSS (Payment Card Industry Data Security Standard) compliance refers to the adherence to a set of security standards designed to protect card information during and after a financial transaction. These standards are established by the Payment Card Industry Security Standards Council (PCI SSC), which was founded by major credit card companies like Visa, MasterCard, American Express, Discover, and JCB.",
+        "services": [
+            {
+                "name": "RDS",
+                "full": "RDS resources",
+                "category": "database"
+            },
+            {
+                "name": "EC2",
+                "full": "EC2 resources",
+                "category": "server"
+            },
+            {
+                "name": "ECS",
+                "full": "ECS Cluster"
+            },
+            {
+                "name": "EKS",
+                "full": "EKS Cluster"
+            },
+            {
+                "name": "API Gateway",
+                "full": "API Gateway"
+            },
+            {
+                "name": "IAM",
+                "full": "IAM Roles and Policies"
+            }
+        ]
+    },
+    {
+        "framework": "ISO 27001",
+        "cloud": "AWS",
+        "description": "ISO 27001 is an internationally recognized standard for managing information security. It provides a framework for establishing, implementing, maintaining, and continually improving an Information Security Management System (ISMS). The standard helps organizations manage and protect their information assets so that they remain secure, using a risk management approach.",
+        "services": [
+            {
+                "name": "RDS",
+                "full": "RDS resources",
+                "category": "database"
+            },
+            {
+                "name": "EC2",
+                "full": "EC2 resources",
+                "category": "server"
+            },
+            {
+                "name": "ECS",
+                "full": "ECS Cluster"
+            },
+            {
+                "name": "EKS",
+                "full": "EKS Cluster"
+            },
+            {
+                "name": "API Gateway",
+                "full": "API Gateway"
+            }
```

Comment:
We are missing IAM, was that intentional or it just not part of the compliance framework?